packages/portfolio-contract/src/planner.exo.ts

```diff
@@ -109,6 +112,20 @@ export const preparePlanner = (
         portfolioPlanner.submitVersion(policyVersion, rebalanceCount);
         portfolioPlanner.resolveFlowPlan(flowId, planOrSteps);
       },
+      rejectPlan(
+        portfolioId: number,
+        flowId: number,
+        reason: string,
+        policyVersion?: number,
+        rebalanceCount?: number,
+      ) {
+        trace('TODO: reject plan', { portfolioId, flowId, reason });
```

Comment:
What will resolve this TODO?

Reply:
I think the TODO I had in mind is now done, so what's left is...

```suggestion
        trace('reject plan', { portfolioId, flowId, reason });
```